Makefile

```diff
@@ -961,6 +961,15 @@ $(foreach board, $(EF_BOARDS), $(eval $(call EF_TEMPLATE,$(board),$($(board)_fri
 # Expand the upgrader rules
 $(foreach board, $(UP_BOARDS), $(eval $(call UP_TEMPLATE,$(board),$($(board)_cpuarch),$($(board)_short))))
 
+bu_playuavosd: bu_playuavosd_px4
+
+.PHONY: bu_playuavosd_px4
+bu_playuavosd_px4: BOARD_ROOT_DIR=$(ROOT_DIR)/flight/targets/playuavosd
+bu_playuavosd_px4: OUTDIR=$(BUILD_DIR)/bu_playuavosd
+bu_playuavosd_px4: bu_playuavosd_tlfw
+	$(V0) @echo "PX4_MKFW    $*"
+	$(V1) $(BOARD_ROOT_DIR)/px_mkfw.py --image $(OUTDIR)/bu_playuavosd.padded.bin --board_id 90 > $(OUTDIR)/bu_playuavosd.px4
```

Comment:
Yes. I wonder if we can easily put something together to go the other way, at least some instructions. I will look into opening a PR against the playuav software to allow it to flash our stuff too